src/iocore/net/OCSPStapling.cc

```diff
@@ -1309,11 +1310,7 @@ ocsp_update()
 
 // RFC 6066 Section-8: Certificate Status Request
 int
-#if !defined(OPENSSL_IS_BORINGSSL) && !defined(OPENSSL_IS_AWSLC)
-ssl_callback_ocsp_stapling(SSL *ssl)
```

Comment:
This only ever compiled thanks to an evil cast:
https://github.com/openssl/openssl/blob/ed7a8bfd7409ac4a516581f1711d98a9362a70d5/include/openssl/tls1.h#L345